_usage/migration.md

```diff
@@ -103,15 +99,15 @@ you can access them with the PostgreSQL protocol as well.  To do that, you can
 use any PostgreSQL client.  Make sure that you connect to the PostgreSQL port
 (by default 5432) and not to the TDS port (by default 1433).
 
-While table and column names will be identical, the names of the schemas in
-PostgreSQL will depend on the
-[migration mode](docs/installation/single-multiple):
+While object names will be identical, the names of the schemas in the migrated SQL
+Server user databases may look different when access through PG, depending on the
+[migration mode_GUC](docs/installation/single-multiple):
```

Comment:
The underscore is in the wrong place, and "GUC" is internal PostgreSQL slang.
"Configuration parameter" would be better.